### DISCUSSION TITLE
Keycodes for Image capturing dont working

### DISCUSSION THREAD

**Discussion by madwurst666** [photobooth-app/photobooth-app#185] [Q&A] [answered]
Hello, the keycodes don't work for me to start the photo box. My system is running on Manjaro Linux. Can anyone help. Thanks Danny

**Comment by mgineer85** [comment, marked as answer]
You followed the installation guide, especially https://photobooth-app.org/setup/installation/#tweak-system-settings-rpi ?
Did you restart afterwards?
Please post your logfile.

**Reply by madwurst666** [reply]
I think i forgot it. I repeat this an the keys „i“ is working thx . Can i use ASCI Keycodes for Input? Like 13 for Return?

**Reply by mgineer85** [reply]
I think it should work. Enable logging on debuglevel , push the key and it will log which key name was pressed. Insert the key name in the textbook and save it.

**Reply by madwurst666** [reply]
Thx for the fast response the key namens „enter“ :) Now it works perfect with my cheap Bluetooth Remote Control from the Smartphone Thank You.
